shell/common/gin_converters/std_converter.h

```diff
@@ -182,6 +186,30 @@ struct Converter<std::map<K, V>> {
   }
 };
 
+#if defined(OS_WIN)
+template <>
+struct Converter<std::wstring> {
+  static v8::Local<v8::Value> ToV8(v8::Isolate* isolate,
+                                   const std::wstring& val) {
+    return Converter<std::u16string>::ToV8(isolate, base::AsString16(val));
+  }
+  static bool FromV8(v8::Isolate* isolate,
+                     v8::Local<v8::Value> val,
+                     std::wstring* out) {
+    if (!val->IsString())
+      return false;
+
+    std::u16string str;
+    if (Converter<std::u16string>::FromV8(isolate, val, &str)) {
+      *out = base::AsWString(str);
```

Comment:
hm... is it OK to use `base::AsWString` here? It looks like `std::wstring` can sometimes be 32-bit characters (though perhaps never in the configurations in which we build Electron currently). It seems like we should use `base::WideToUTF16` anyway here to be on the safe side. It should be the same performance-wise when wchar_t is 16-bits.

Reply:
Making `str` a wstring here and using `Converter<std::wstring>::FromV8()` on line 203  would be problematic since block this is the implementation of Converter<std::wstring>::FromV8() :slightly_smiling_face: